internal/sympath/walk.go

```diff
@@ -69,7 +67,8 @@ func symwalk(path string, info os.FileInfo, walkFn filepath.WalkFunc) error {
 	if IsSymlink(info) {
 		resolved, err := filepath.EvalSymlinks(path)
 		if err != nil {
-			return errors.Wrapf(err, "error evaluating symlink %s", path)
+			log.Printf("Skipping broken symlink: %s", path) // Log broken symlink
+			return nil
```

Comment:
I have three thoughts about this change...

1. This doesn't just change the behavior for JUST .helmignore files. It changes it for everything. Even those that are not ignored.
2. There is a situation where a broken symlink would have stopped rendering because of bad output and this change will allow for bad output because the broken symlink will be ignored and movement will continue. This could cause cases for an accidental bad deployment.
3. This brings in the ability to easily package broken symlinks. There are security implications to using symlinks as they can be easily abused and this makes including many broken ones (on the packagers system) possible.

I think there needs to be a better route where it's more directly linked to the helmignore feature rather than general walking.

Reply:
I did some more digging and am able to reproduce this. Will need to do some more testing to make sure there are no security issues with it AND to make sure nothing breaks.